cluster.yml

```diff
@@ -34,59 +34,59 @@
       when: "'rkt' in [etcd_deployment_type, kubelet_deployment_type, vault_deployment_type]"
 
 - hosts: etcd:k8s-cluster:vault
-  any_errors_fatal: true
+  any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
   roles:
     - { role: kargo-defaults, when: "cert_management == 'vault'" }
     - { role: vault, tags: vault, vault_bootstrap: true, when: "cert_management == 'vault'" }
 
 - hosts: etcd
-  any_errors_fatal: true
+  any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
```

Comment:
our setup establishes quorum based on initial members (all of etcd group). there's no chance of data corruption unless you restart hosts. in any case, failure is failure and you should retry.